Name change (repeat Patch 1.1 to have pull request)

Dom-Bach opened the pull request:
No change, just a pull request.

Before that was Patch 1.1:
-(skeleton) Code of PsychoPy masking experiment with the most important functions
-Provide example stimuli

Next:
-Make code run properly
-Clean code according to style guide

Dom-Bach committed:
Name change (repeat Patch 1.1 to have pull request)

No change, just a pull request.

Before that was Patch 1.1:
-(skeleton) Code of PsychoPy masking experiment with the most important functions
-Provide example stimuli

Next:
-Make code run properly
-Clean code according to style guide

Dom-Bach commented:
I accidentally pushed changes instead of creating a pull request, so I made a meaningless change just so that I have a pull request.

vandenman commented:
For next time, please first create a feature branch and only work on that branch. Afterward, make a pull request. The only thing the reviewers can now see and comment on is the "meaningless change", so that means all the tools GitHub has for code review cannot be used and makes it difficult for them give good feedback.

For the reviewers, if you click [here](https://github.com/Programming-The-Next-Step/Masking_experiment/commit/eab472044b79a2a74b1d97d27932898f2fcf09da) you get an overview of the changes in the last commit, please add your comments there.

Dom-Bach commented:
> For next time, please first create a feature branch and only work on that branch. Afterward, make a pull request. The only thing the reviewers can now see and comment on is the "meaningless change", so that means all the tools GitHub has for code review cannot be used and makes it difficult for them give good feedback.
> 
> For the reviewers, if you click [here](https://github.com/Programming-The-Next-Step/Masking_experiment/commit/eab472044b79a2a74b1d97d27932898f2fcf09da) you get an overview of the changes in the last commit, please add your comments there.

Thanks for fixing it, will do!

Dom-Bach committed:
Patch 1.2: Maturation of Code

+Many fixes to the code
-closing of infinitive loops
-fixing the main for-loop
-welcome screen text works (and code for it is more elegant)
-including missing statements (e.g., win.flip() )
-logging fixed
-> code runs now

+Stylistic improvements:
-redundant parts of boolean statements removed
-spaces between operators added

+New features:
-on localization tasks, masks are shown on both possible locations, so that they don't give away the location, on trials without localization, mask is only shown centrally (like the target)
-new boolean "detec_skip": If participants say they don't see a stimulus, should followup tasks (that address the stimulus) be skipped?
-Experiment can now be quit at any point in time, not only between trials
-Logging now picks up when tasks are skipped or when the experiment is quit

Dom-Bach commented:
This week's changes can be seen in "Patch 1.2".
Didn't wanna delete the earlier patch, as I'm not sure what would happen to the earlier feedback.

fdabl commented:
@Dom-Bach there is no branch "Patch 1.2" as far as I can see. But you did make a commit yesterday on the branch "Dom-Bach-patch-1". Should this be reviewed? Then please make a new pull request!

Once this is done, @merope95 and @carolinstr please review the new pull request. If @Dom-Bach does not make a new pull request, please consider this review request void.

Dom-Bach commented:
> @Dom-Bach there is no branch "Patch 1.2" as far as I can see. But you did make a commit yesterday on the branch "Dom-Bach-patch-1". Should this be reviewed? Then please make a new pull request!
> 
> Once this is done, @merope95 and @carolinstr please review the new pull request. If @Dom-Bach does not make a new pull request, please consider this review request void.

@fdabl Yeah, the changes were the commit to that branch. I think I fixed it now?!
https://github.com/Programming-The-Next-Step/Masking_experiment/pull/6#issue-421293387

fdabl commented:
Yes @Dom-Bach, I now see another pull request!

princessamygdala reviewed (commented):
i did this review already in the beginning but out of some reason the review request did not vanish but still remained to be pending so ill just right now submit this comment to get it out of my list to review stuff :)
